README.md

```diff
@@ -1,6 +1,6 @@
 # Composite Resource Module
 
-[![Build status](https://ci.appveyor.com/api/projects/status/c80a8uja31avfha4?svg=true)](https://ci.appveyor.com/project/mgreenegit/compositeresource)
+[![Build status](https://ci.appveyor.com/api/projects/status/c80a8uja31avfha4/branch/master?svg=true)](https://ci.appveyor.com/project/Microsoft/CompositeResource)
```

Comment:
Changed the link to `https://ci.appveyor.com/api/projects/status/c80a8uja31avfha4/branch/master?svg=true` (using branch master), and that is showing passed. I assume the URL was wrong, and it was the correct token

[![Build status](https://ci.appveyor.com/api/projects/status/c80a8uja31avfha4/branch/master?svg=true)](https://ci.appveyor.com/project/Microsoft/CompositeResource)